x-pack/plugins/maps/server/saved_objects/maps_telemetry.ts

```diff
@@ -37,8 +37,6 @@ export const mapsTelemetrySavedObjects: SavedObjectsType = {
               avg: { type: 'long' },
             },
           },
-          layerTypesCount: { dynamic: 'true', properties: {} },
-          emsVectorLayersCount: { dynamic: 'true', properties: {} },
```

Comment:
Would it be enough if we did `layerTypesCount: { type: 'object' }`? That would allow us storing the object but not indexing each value in it (since we are not actually searching/aggregating for those values)

Reply:
That might make more sense here as a placeholder since we do want to retain some version of these moving forward but without using dynamic fields. I've added back in the logic supporting construction of these fields and changed their types per your recommendation.

Reply:
we also need to set `enabled: false` to prevent these from being indexed. If we're not querying this telemetry data from Kibana, can we just set the entire maps-telemetry mapping to `type: 'object', enabled: false`?

Reply:
@rudolf Per the conversation above, I've actually removed this saved object. It appears to be leftover from older logic and is no longer required!